stdlib/public/SwiftShims/SwiftStddef.h

```diff
@@ -13,7 +13,12 @@
 #ifndef SWIFT_STDLIB_SHIMS_SWIFT_STDDEF_H
 #define SWIFT_STDLIB_SHIMS_SWIFT_STDDEF_H
 
+#if !defined(__clang__)
+#include <stdint.h>
+typedef size_t __swift_size_t;
```

Comment:
The purpose of these headers is to get system-equivalent types without importing any system headers, for ordering reasons. If MSVC doesn't have something like `__SIZE_TYPE__`, it's better just to use an appropriate raw type, and maybe we can static_assert somewhere else that it's the same.